Controller/ExpiredVideoController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Pumukit\ExpiredVideoBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/admin/expired/video")
+ */
+class ExpiredVideoController extends Controller
+{
+    private $days = 365;
+    private $roleCod = "owner";
```

Comment:
@Yurujai, @rubenrua 
This value should be read from `'pumukitschema.personal_scope_role_code'` parameter:
https://github.com/campusdomar/PuMuKIT2/blob/2.3.x/src/Pumukit/SchemaBundle/DependencyInjection/Configuration.php#L57-L60
https://github.com/campusdomar/PuMuKIT2/blob/2.3.x/src/Pumukit/SchemaBundle/DependencyInjection/PumukitSchemaExtension.php#L33